memory_viz.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script type="module" src="https://cdn.jsdelivr.net/gh/pytorch/pytorch@main/torch/utils/viz/MemoryViz.js"></script>
```

Comment:
Why do you need different CDN than main github?
```suggestion
<script type="module" src="https://raw.githubusercontent.com/pytorch/pytorch/main/torch/utils/viz/MemoryViz.js"></script>
```

Reply:
When served from the `raw` endpoint it has the wrong MIME type: `TypeError: 'text/plain' is not a valid JavaScript MIME type.`. This is why things like `jsdelivr` were  created which just serve text files from GitHub as javascript files.